libexec/rtld-elf/aarch64/reloc.c

```diff
@@ -377,7 +377,10 @@ tramp_pgs_append(uintptr_t target, const Obj_Entry *dst)
 	memcpy(t, template, len);
 	// Arm64's i-cache and d-cache are not coherent. So we need to clean the d-cache
 	// and invalidate the i-cache. See https://community.arm.com/arm-community-blogs/b/architectures-and-processors-blog/posts/caches-and-self-modifying-code
-	__clear_cache(&t->padding, (char *)t + len);
+	// We derive the start and end addresses from pg so that they are aligned to the
+	// cache-line size, since pg is page aligned.
+	__clear_cache(cheri_copyaddress(pg, t->code),
+		      cheri_copyaddress(pg, (uintptr_t)t + len));
```

Comment:
Four spaces for continuation lines rather than aligning with the first argument.